samwise.js

```diff
@@ -1,15 +1,14 @@
 /**
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.